src/Components/Button/index.jsx

```diff
@@ -0,0 +1,16 @@
+import'./styles.css'
+
+const Button = ({ children, onClick }) => {
+  return (
+    <div className='container-button'>
+      <button
+        className="component-button"
+        onClick={onClick}
+      >
+        {children}
+      </button>
+    </div>
+  )
```

Comment:
Debería ir solo el button, el DIV sobra...
No se le debe dar un contenedor acá, sino que el contenedor se crear y se mete este componente donde se vaya a usar.
la etiqueta div con className container button debe ponerse en el login, junto con sus estilos...y eliminarse de este componente.

Reply:
en login in quedaría así

``` 
<div className='container-button'>
   <Button
       onClick={() => handleSession()}
       children={"Ingresar"}
    />
</div> ```